addons/website_slides_survey/views/slide_channel_views.xml

```diff
@@ -5,10 +5,19 @@
         <field name="model">slide.channel</field>
         <field name="inherit_id" ref="website_slides.view_slide_channel_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//span[@name='members_completed_count_label']" position="replace">
+            <xpath expr="//button[@name='action_redirect_to_completed_members']" position="after">
                 <field  name="nbr_certification" invisible="1"/>
-                <span class="o_stat_text" invisible="nbr_certification &gt; 0">Finished</span>
-                <span class="o_stat_text" invisible="nbr_certification == 0">Certified</span>
+                <button name="action_redirect_to_certified_members"
+                    invisible="nbr_certification == 0"
+                    type="object"
+                    icon="fa-trophy"
+                    class="oe_stat_button"
+                    groups="website_slides.group_website_slides_officer">
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value"><field name="members_certified_count" nolabel="1"/></span>
+                        <span name="members_certified_count_label" class="o_stat_text">Certified</span>
```

Comment:
Last small detail, the name on this span is actually not necessary, I'm pretty sure that the "members_completed_count_label" name was added in order to be easily overridable in here, and is now also kind of useless.

Not a big deal though, but though I'd mention it 🙂